Project.toml

```diff
@@ -17,6 +17,7 @@ Cbc = ">= 0.4"
 InfrastructureModels = "^0.2"
 Ipopt = ">= 0.4"
 JuMP = "^0.19.1"
+JSON = ">= 0.20"
```

Comment:
I think this bound can be v0.18.  I think that is when they added julia v1.0 support.